src/components/Picker/Picker.styles.ts

```diff
@@ -25,5 +36,35 @@ export const getClassNames = memoizeFunction((classNameProps: PickerClassNamePro
     suggestionItem: {
       width: '100%',
     },
+    suggestionsBox: {
+      selectors: {
+        '.ms-Suggestions-title': {
+          color: theme.palette.themePrimary,
+          borderBottomColor: theme.palette.neutralLight,
+        },
+        '.ms-Suggestions-container': {
+          borderBottomColor: theme.palette.neutralLight,
+        },
+      },
+    },
+    suggestionsList: {
+      selectors: {
+        '.ms-Suggestions-item': {
+          selectors: {
+            ':hover': {
+              backgroundColor: theme.palette.neutralLighter,
+            },
+          },
+        },
+        '.ms-Suggestions-item[class*="suggestionsItemIsSuggested"]': {
+          backgroundColor: theme.palette.neutralLight,
+          selectors: {
+            ':hover': {
+              backgroundColor: theme.palette.neutralTertiaryAlt,
+            },
+          },
+        },
+      },
+    },
```

Comment:
I think there should be a way to do this cleaner by using mergeStyles's parent/child referencing features:
https://github.com/OfficeDev/office-ui-fabric-react/wiki/mergeStyles#referencing-child-elements-within-the-mergestylesets-scope

I gave it a first shot and the selectors stopped working. I'll try to revisit later.

Reply:
i'm not thrilled with it either, but it don't see an obvious better way without a refactor, or the upstream Picker getting converted to mergeStyles first. let's leave this one as-is because we'll soon be removing all of our Fabric style overrides anyway